src/Workload/Microsoft.Maui.Controls.Foldable.Runtime/Microsoft.Maui.Controls.Foldable.Runtime.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+  <Import Project="../Shared/Common.targets" />
+  <Import Project="../Shared/Frameworks.targets" />
+
+  <PropertyGroup>
+    <Description>.NET MAUI Controls Foldable runtime pack</Description>
+    <OutputPath Condition=" '$(MauiPlatformName)' != '' ">$(DotNetPacksDirectory)$(PackageId)/$(PackageVersion)/</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="@(_TargetPlatform->'$(MauiRootDirectory)src/Compatibility/Core/src/bin/$(Configuration)/%(Tfm)/Microsoft.Maui.Controls.Compatibility.dll')" />
+    <None Include="@(_TargetPlatform->'$(MauiRootDirectory)src/Compatibility/Core/src/bin/$(Configuration)/%(Tfm)/Microsoft.Maui.Controls.Compatibility.pdb')" />
+    <None Include="@(_TargetPlatform->'$(MauiRootDirectory)src/Controls/src/Core/bin/$(Configuration)/%(Tfm)/Microsoft.Maui.Controls.dll')" />
+    <None Include="@(_TargetPlatform->'$(MauiRootDirectory)src/Controls/src/Core/bin/$(Configuration)/%(Tfm)/Microsoft.Maui.Controls.pdb')" />
+    <None Include="@(_TargetPlatform->'$(MauiRootDirectory)src/Controls/src/Xaml/bin/$(Configuration)/%(Tfm)/Microsoft.Maui.Controls.Xaml.dll')" />
+    <None Include="@(_TargetPlatform->'$(MauiRootDirectory)src/Controls/src/Xaml/bin/$(Configuration)/%(Tfm)/Microsoft.Maui.Controls.Xaml.pdb')" />
```

Comment:
If this pack is for a new assembly, only the 1 assembly should be inside? When using this pack you'll get all the other MAUI assemblies already.

Reply:
Does that look better?